automatic/jenkins/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,105 @@
+$ErrorActionPreference = 'Stop'
+
+# Set all passed package parameters as variables
+(Get-PackageParameters).GetEnumerator().ForEach{
+    Set-Variable -Name $_.Key -Value $_.Value
+}
```

Comment:
I honestly don't like this approach.

Instead of polluting the current space with variables that don't need to be set in that way, it should probably be changed to do `$pp = Get-PackageParameters`, which is the convention other packages use.

Reply:
I've limited the scope of this, though I'm now very curious about what parameters are in Get-PackageParameters that aren't parameters-passed-in-for-use. I'll have to have a play sometime!